lib/matplotlib/backends/backend_pdf.py

```diff
@@ -183,12 +183,15 @@ def _create_pdf_info_dict(backend, metadata):
     info = {k: v for (k, v) in info.items() if v is not None}
 
     def is_string_like(x):
+        """an instance of str"""
```

Comment:
```suggestion
        """Return whether *x* is an instance of ``str``."""
```

Reply:
No, the point of the docstring is to be used in the error message. It's a nested function in a private method, so is not otherwise user-facing.

Reply:
Oh! In that case, please add a comment below the docstring. Otherwise somebody will "clean" this up later. I'm also wondering why flake8-docstrings is not complaining about the non-compliant docstring format.

Reply:
Thinking about it, using the docstring as a message is too clever.

We should instead use an explicit attribute.

~~~
def is_string_like(x):
    return isinstance(x, str)
is_string_like.expected_value = "an instance of str"
~~~